experimental/skills/DialogToDialog/DialogRootBot/Cards/welcomeCard.json

```
@@ -0,0 +1,128 @@
+﻿{
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "type": "AdaptiveCard",
+  "version": "1.0",
+  "body": [
+    {
+      "type": "TextBlock",
+      "spacing": "Medium",
+      "size": "Medium",
+      "weight": "Bolder",
+      "text": "Welcome to the Dialog Skill Prototype!",
+      "wrap": true,
+      "maxLines": 0,
+      "color": "Accent"
+    },
+    {
+      "type": "TextBlock",
+      "size": "default",
+      "text": "Here are some things you can do:",
+      "wrap": true,
+      "maxLines": 0
+    },
+    {
+      "type": "TextBlock",
+      "size": "default",
+      "text": "Send a text message to the bot (message, single-turn)",
+      "wrap": true,
+      "weight": "Bolder",
+      "color": "Accent"
+    },
+    {
+      "type": "TextBlock",
+      "size": "default",
+      "text": "**m:some message** sends the text after **m:** directly to the skill and the skill does intent and entity recognition against LUIS.",
+      "wrap": true,
+      "spacing": "None"
+    },
+    {
+      "type": "TextBlock",
+      "size": "default",
+      "text": "Send a BookFlight event (event, multi-turn)",
+      "wrap": true,
+      "weight": "Bolder",
+      "color": "Accent"
+    },
+    {
+      "type": "TextBlock",
+      "size": "default",
+      "text": "**bookflight** sends an event to the skill with the name = 'BookFlight' to start a flight booking dialog.",
+      "wrap": true,
+      "spacing": "None"
+    },
+    {
+      "type": "TextBlock",
+      "size": "default",
+      "text": "Send a GetWeather invoke activity (invoke, single-turn)",
+      "wrap": true,
+      "weight": "Bolder",
+      "color": "Accent"
+    },
+    {
+      "type": "TextBlock",
+      "size": "default",
+      "text": "**getweather** sends an invoke activity to the skill with the name = 'GetWeather'. The skill replies right away (Invoke activities are Request/Response).",
+      "wrap": true,
+      "spacing": "None"
+    },
+    {
+      "type": "TextBlock",
+      "size": "default",
+      "text": "OAuthTest (event, multi-turn)",
+      "wrap": true,
+      "weight": "Bolder",
+      "color": "Accent"
+    },
+    {
+      "type": "TextBlock",
+      "size": "default",
+      "text": "**oauthtest** sends an event to the skill to start an OAuthCard dialog.",
+      "wrap": true,
+      "spacing": "None"
+    },
+    {
+      "type": "TextBlock",
+      "size": "default",
+      "text": "Send a message to the bot with values (message, single-turn)",
+      "wrap": true,
+      "weight": "Bolder",
+      "color": "Accent"
+    },
+    {
+      "type": "TextBlock",
+      "size": "default",
+      "text": "**mv:some message** sends some complex object in the Value property of the activity in addition to the message after **mv:**.",
+      "wrap": true,
+      "spacing": "None"
+    },
+    {
+      "type": "TextBlock",
+      "size": "default",
+      "text": "Cancelling multi-turn conversations",
```

While both spellings are correct, our style guide suggests the version with one "l".
```suggestion
      "text": "Canceling multi-turn conversations",
```

---

Updated text

---
In reply to: [356198748](https://github.com/microsoft/BotBuilder-Samples/pull/2032#discussion_r356198748) [](ancestors = 356198748)